mxcube/Dockerfile

```diff
@@ -5,6 +5,8 @@ RUN apt-get update
 RUN apt-get -y install \
     # need for lucid3
     libgl1-mesa-glx \
+    # for video streamer
+    ffmpeg \
```

Comment:
You could also add it as a *conda* dependency.

This is what is going to happen upstream: <https://github.com/mxcube/mxcubeweb/pull/1504>.

Reply:
Done